src/SearchCards.js

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from "react"
+import { Link } from "react-router-dom"
+import axios from "axios"
+
+const SearchCard = () => {
+
+	const [magicCards, setMagicCards] = useState([])
+
+	useEffect(() => {
+		axios({
+			url: "https://api.magicthegathering.io/v1/cards",
+			params: {
+                pageSize: 5,
+                random: true,
+			}
+		}).then((allMagicCards) => {
+			setMagicCards(allMagicCards.data.cards)
+		})
+	}, [])
+
+	return (
+		<ul className="catalogue">
+			{magicCards.map((cardsObj) => {
+				return (
+					<li key={cardsObj.id}>
+						<Link to={`/cards/${cardsObj.id}`}>
+							<img src={cardsObj.imageUrl} alt={cardsObj.name} />
```

Comment:
you can setup an onError catch to have a placeholder image if a card doesn't load or if there's issues!

Reply:
Thats a great suggestion and was something I was playing with but came up with a different work around. Thanks!